### PR TITLE
add tAutocomplete@1.0.0 w/ git autoupdate

### LINKED ISSUE #8697: [Request] Add tAutocomplete
**Library name:** vyasrao
**Git repository url:**
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8697 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/vyasrao/tAutocomplete
 * Official website (optional, not the repository):https://vyasrao.github.io/tAutocomplete/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:48
   - Count of watchers:9
   - Count of forks:16
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8697, cc @vyasrao

### CODE REVIEW

File: ajax/libs/tAutocomplete/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "jquery": ">=1.5"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/vyasrao/tAutocomplete.git",
+    "basePath": "",
+    "files": [
+      "tautocomplete*.!(*json|html)",
```
Comment (PeterDaveHello) [inline ajax/libs/tAutocomplete/package.json]: There is no html file with `tautocomplete` name leading.

**Comment by pvnr0082t** [comment]
@x09326 I have fixed filemap. Please help me review again, thank you.

File: ajax/libs/tAutocomplete/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "jquery": ">=1.5"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/vyasrao/tAutocomplete.git",
+    "basePath": "",
+    "files": [
+      "tautocomplete*.!(*json)",
```
Comment (PeterDaveHello) [inline ajax/libs/tAutocomplete/package.json]: Can you make sure there is a dot before json? Since you look like going to exlude the json file, right?

Comment (pvnr0082t) [inline ajax/libs/tAutocomplete/package.json]: You're right. But I tried `tautocomplete*!(*json)` and `tautocomplete*!(json)` and these two will grab `tautocomplete.jquery.json`.

Comment (PeterDaveHello) [inline ajax/libs/tAutocomplete/package.json]: Does this work?
`"tautocomplete*.!(*.json)",`

Comment (pvnr0082t) [inline ajax/libs/tAutocomplete/package.json]: I tried `"tautocomplete*.!(*.json)"` but it also grabbed `tautocomplete.jquery.json`. But `tautocomplete!(*.json)` can work.

**Comment by pvnr0082t** [comment]
@x09326 I have fixed filemap. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Pull request merged**